cockatrice/src/player.cpp

```diff
@@ -2685,6 +2686,28 @@ void Player::updateCardMenu(const CardItem *card)
             cardMenu->addAction(aClone);
         }
     }
+    addRelatedCardView(card, cardMenu);
+}
+
+void Player::addRelatedCardView(const CardItem *card, QMenu *cardMenu)
+{
+    if (card == nullptr || cardMenu == nullptr || card->getInfo() == nullptr) {
+        return;
+    }
+
+    QList<CardRelation *> relatedCards = card->getInfo()->getRelatedCards();
+    if (relatedCards.isEmpty()) {
+        return;
+    }
+
+    cardMenu->addSeparator();
+    auto *signalMapper = new QSignalMapper(this);
+    for (const CardRelation *relatedCard : relatedCards) {
+        QAction *viewCard = cardMenu->addAction("Show Card: \"" + relatedCard->getName() + "\"");
```

Comment:
`...->addAction(tr("View Card") + ":\"" + relatedCard->getName() + "\"");`

Reply:
Note especially the `tr`, that's important so the string gets translated for other languages